examples/test_util/EclIO.cpp

```diff
@@ -0,0 +1,1179 @@
+}
+
+
+template <class T> std::string EclIO::make_scientific_string(T number){
+
+    std::stringstream buffer;
+    std::string str1,str2,exStr;
+    int p1,exp;
+    bool isDouble=false;
+    
+    if (sizeof(number)==8){
+        isDouble=true;
+    }
+    
+    if (isDouble==true){
+        buffer << std::uppercase << std::scientific << std::setprecision(13) << number << std::nouppercase;
+        exStr="D";
+    } else {
+        buffer << std::uppercase << std::scientific << std::setprecision(7) << number << std::nouppercase;
+        exStr="E";
+    }
+    
+    str1=buffer.str();
+    p1=str1.find_first_of("E");
+    
+    str1=str1.substr(0,p1);
+    p1=str1.find_first_of(".");
+    str1.erase(p1,1); 
+    str1.insert(p1-1,"0."); 
+
+    str2=buffer.str();
+
+    p1=str2.find_first_of("E");
+    str2=str2.substr(p1+1,str2.length()-p1-1);
+
+    exp=stoi(str2)+1;
+    
+    if (abs(exp)>99){
+       exStr="";
+    }
+    
+    buffer.str("");
+
+    if (abs(exp)<10){
+        if (exp < 0){
+	    buffer << str1 << exStr << "-0" << abs(exp); 
+        } else {
+	    buffer << str1 << exStr << "+0" << abs(exp); 
+        }
+    } else {
+        if (exp>0){
+	    buffer << str1 << exStr << "+" << exp; 
+        } else {
+	    buffer << str1 << exStr << exp; 
+	}    
+    }
+
+    return buffer.str();
+}
+
+
+void EclIO::readBinaryHeader(std::fstream &fileH,std::string &arrName,int &ant,std::string &arrType){
+
+    /* header data 4 byte (value 16), 8 byte (name of array), 4 byte number of elements, 4 byte array type 
+     ends with 4 byte (value should be 16 */
+
+    int bhead;
+    std::string tmpStrName(8,' ');
+    std::string tmpStrType(4,' ');
+
+    if (!fileH.is_open()){
+        std::string message="fstream fileH not open for reading";
+        throw std::runtime_error(message);
+    }
+
+    fileH.read((char *)&bhead, sizeof(bhead));
+    bhead=reverseInt(bhead);
+    
+    if (bhead!=16){
+        std::string message="Error reading binary header. Expected 16 bytes of header data, found " + std::to_string(bhead);
+        throw std::runtime_error(message);
+    }
+    
+    fileH.read(&tmpStrName[0], 8);
+    
+    fileH.read((char *)&ant, sizeof(ant));
+    ant=reverseInt(ant);
+
+    fileH.read(&tmpStrType[0], 4);
+
+    fileH.read((char *)&bhead, sizeof(bhead));
+    bhead=reverseInt(bhead);
+    
+    if (bhead!=16){
+        std::string message="Error reading binary header. Expected 16 bytes of header data, found " + std::to_string(bhead);
+        throw std::runtime_error(message);
+    }
+    
+    arrName=tmpStrName;
+    arrType=tmpStrType;
+}
+
+
+void EclIO::stepOverArray(std::fstream &fileH,const int ant, const std::string arrType){
+
+    long int step=0;
+   
+    if (arrType=="INTE"){
+       
+        step=ant*4;
+        step=step + ((ant-1) / 1000)*8;    // ignore 8 byte every 1000 value        
+       
+        if (ant>0){
+            step=step+8;
+	}
+	
+    } else if (arrType=="CHAR"){
+        step=ant*8;
```

Comment:
OPM is an english codebase, you have many variables which have names inspired by short forms of Norwegian nouns - use English!

Reply:
fixed ant -> num